src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocksNoViolations.java

```diff
@@ -5,99 +5,92 @@
 
 import java.io.IOException;
 
-public class InputEmptyBlocksAndCatchBlocksNoViolations
-{
-    private void foo6() {
-        try {
-            throw new IOException();
-        } catch (IOException expected) { // This is expected
-            int k = 0;
-        }
+public class InputEmptyBlocksAndCatchBlocksNoViolations {
+  private void foo6() {
+    try {
+      throw new IOException();
+    } catch (IOException expected) { // This is expected
+      int k = 0;
     }
+  }
 
-    public void testTryCatch()
-    {
-        try {
-            int y=0;
-            int u=8;
-            int e=u-y;
-            return;
-        }
-        catch (Exception e) {
-            System.identityHashCode(e);
-            return;
-        }
-        finally
-        {
-            return;
-        }
+  public void testTryCatch() {
+    try {
+      int y = 0;
+      int u = 8;
+      int e = u - y;
+      return;
+    } catch (Exception e) {
+      System.identityHashCode(e);
+      return;
+    } finally {
+      return;
     }
+  }
 
-    public void testTryCatch3()
-    {
-        try {
-            int y=0;
-            int u=8;
-            int e=u-y;
-        }
-        catch (IllegalArgumentException e) {
-            System.identityHashCode(e); //some comment
-            return;
-        }
-        catch (IllegalStateException ex) {
-                System.identityHashCode(ex);
-                return;
-        }
+  public void testTryCatch3() {
+    try {
+      int y = 0;
+      int u = 8;
+      int e = u - y;
+    } catch (IllegalArgumentException e) {
+      System.identityHashCode(e); // some comment
+      return;
+    } catch (IllegalStateException ex) {
+      System.identityHashCode(ex);
+      return;
     }
+  }
 
-    public void testTryCatch4()
-    {
-        int y=0;
-        int u=8;
-        try {
-            int e=u-y;
-        }
-        catch (IllegalArgumentException e) {
-            System.identityHashCode(e);
-            return;
-        }
+  public void testTryCatch4() {
+    int y = 0;
+    int u = 8;
+    try {
+      int e = u - y;
+    } catch (IllegalArgumentException e) {
+      System.identityHashCode(e);
+      return;
     }
-    public void setFormats() {
-        try {
-            int k = 4;
-        } catch (Exception e) {
-            Object k = null;
-            if (k != null)
-                k = "ss";
-            else {
-                return;
-            }
-        }
+  }
+
+  public void setFormats() {
+    try {
+      int k = 4;
+    } catch (Exception e) {
+      Object k = null;
+      if (k != null) k = "ss";
```

Comment:
Why was this changed and does our google config agree? (new line removed)

Reply:
formatter changed it and I tried checking the code against `verifyWithWholeConfig`, it was giving `61:7: 'if' construct must use '{}'s.` error for line 61

Reply:
this could a bug in google formatter maybe? Cuz it is mentioned in style guide that we must use `{}`.

documentation link: https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s4.1.1-braces-always-used

Reply:
Attention to `Braces are used`. It never says braces are optional. This looks like a bug in the formatter unless some other specific rule in the style guide says.

Reply:
I guess there isnt other rule which enforces to not use braces

Reply:
https://google.github.io/styleguide/javaguide.html#s4.1-braces
 
I pretty sure they use this line

Reply:
but finishing discussion on spacing should not block this PR.

Reply:
It seems the only way to get things attention is to block PRs. It has been mentioned in discord something was an issue days ago. I have not seen the issue created for it. I question if we are just doing things blindly and not doing any real analysis. Quality should matter more than just getting the word done.

Reply:
> @Zopsss , please open issue formatter to get 100% clarification.

I did created test cases for the highlighted part in one of my old PR. Here's that PR: https://github.com/checkstyle/checkstyle/pull/14912

We're following the rule properly and testing every cases of it

Reply:
> It seems the only way to get things attention is to block PRs. It has been mentioned in discord something was an issue days ago. I have not seen the issue created for it. I question if we are just doing things blindly and not doing any real analysis. Quality should matter more than just getting the word done.

Opened a new issue for `{` placement as you requested: https://github.com/checkstyle/checkstyle/issues/15324 @rnveach

Reply:
@rnveach, this is known issue in formatter - https://github.com/google/google-java-format/issues/51

4.1.3 section should not care about this and we are ok to reuse auto format there.

Reply:
@Zopsss  please add to checkstyle/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule411optionalbracesusage/InputUseOfOptionalBraces.java `if (k != null) k = "ss";` , in separate PR,  to triple confirm that violation still there.

Reply:
done https://github.com/checkstyle/checkstyle/pull/15333/

Reply:
Merged

Reply:
This still needs a response.

Reply:
@rnveach , 
I am not sure what is left unanswered.

We found issue https://github.com/google/google-java-format/issues/51 it is known defect 

And we checkstyle violate this code, it is good - confirmed in that issue.

Reply:
That is what I was looking for which wasn't provided.